README.md

```diff
@@ -111,7 +111,7 @@ vulkan_samples test bonza --headless
 vulkan_samples batch --category performance --duration 10
 
 # Run Swapchain Images sample on an Android device
-adb shell am start-activity -n com.khronos.vulkan_samples/com.khronos.vulkan_samples.SampleLauncherActivity -es "cmd" "sample swapchain_images"
+adb shell am start-activity -n com.khronos.vulkan_samples/com.khronos.vulkan_samples.SampleLauncherActivity -e sample swapchain_images
```

Comment:
Does this still require `""` -> `-e "sample swapchain_images"`

Reply:
I think it's okay without the "".  At least until someone creates a sample name without printable characters in it or something strange like that.